---
layout: default_2
title: Research
permalink: research.html
background: '/img/bg-research.jpeg'
---

<!-- Introduction -->
<section id="research" style="padding-top: 30px; padding-bottom:25px;">
	<div class = "container">
		<div class="row">
			
		    <h4 style= "color: #000000; text-transform: none; font-weight: 350;"> My research broadly encompasses evolution and ecology of flowering plants, with a specific focus on genome size diversity in the form of whole-genome duplication, or polyploidy. As an plant evolutionary and computational biologist, I am able to implement a holistic approach to understand the consequences of whole genome duplication across temporal and spatial scales.  I am also passionate about using natural history collections in research and education.</h4> 

		</div>
	</div>
</section>

<!-- Polyploidy Evolution -->
<section id="polyploidy evolution" style="padding-top: 20px; padding-bottom: 30px;">
	<div class = "container">
		<div class = "row">
   	    	<div class="caption" style="text-align:left; font-size: 22px; line-height: 1.5; color: #fa4616;"><b>Whole Genome Duplication</b></div>
    		<hr style = "margin-top: 10px; margin-bottom: 10px; border-top-width: 2px;"> </hr>
    		
		    <h4 style= "color: #000000; text-transform: none; font-weight: 350;"> Whole genome duplication is when additional copies of all chromosomes occur. Instead of having 2 copies of each chromosome (like humans), polyploids may have 3 or more copies of each chromosome. The extra set of chromosomes can be inherited from the same individual—this is known as autopolyploidy. </h4>		    		
			<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> Polyploidy is known to play an important role in the evolution and diversification of vascular plants. So far, most research has shown that whole genome duplication alone does not lead to a single set of expectations (other than maybe the nucleotypic effect), instead there are dynamic ecological or evolutionary consequences. The search for general patterns or trends associated with polyploidy events still remains – and this is a particularly interesting and fun topic to pursue. </h4>


		
	<!-- Mixed-ploidy species -->	   		
	<div class = "row">
	<div class="col-md-8">
	<h4  style= "color: #000000; text-transform: none; font-weight: 350;"><a style = "color: #fa4616;"><b>Population Ecology and Genetics</b> </a> </h4>
		<h4  style= "color: #000000; text-transform: none; font-weight: 350;">My dissertation focused on investigating coexistence of multiple cytotypes. Historically, it was believed that a tetraploid had to outcompete its diploid progenitor to persist. 
		The idea was that a tetraploid formed, spread, and caused the diploid to be geographically restricted and rare (Stebbins, 1947, 1971). We now know that closely related 
		polyploids and their diploid progenitors can both remain extant. </h4>
		<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> <a style = "color: #fa4616;"><b>Theory:</b> </a>  To understand stability of coexisting cytotypes, 
		I designed a new matrix population model with demographic and environmental stochasticity. Here, I identified that stable coexistence is very probable among mixed-cytotypes,
		 even when reproductive isolation is absent.  See our <a href="https://doi.org/10.1101/2023.03.29.534764" target="_blank">preprint available</a> and our R package,
		  <a href="https://mlgaynor.com/AutoPop/" target="_blank">AutoPop</a>. To find out more about this project, check out my <a href="https://lnkd.in/eGtJfBuQ" target="_blank">2024 Polyploidy Webinar.</a> </h4>
		<h4 style= "color: #000000; text-transform: none; font-weight: 350;">  The genetic consequences of mixed-cytotype
		 populations with ongoing gene flow among cytotypes are mostly unknown. 
		 Therefore, this model has been extended to the Modern Synthesis with 
		 the incorporation of genotypes, mutation, and more. To find out more about our genetic model check out 
		 my <a href="https://youtu.be/mEvSgXN6jbs?t=1990" target="_blank"> 2021 Polyploidy Webinar.</a> </h4>
		 <h4 style= "color: #000000; text-transform: none; font-weight: 350;"> <a style = "color: #fa4616;"><b>Nature:</b></a> Much of my population-level work has focused 
			on autopolyploid <i>Galax urceolata</i> (Diapensiaceae), which includes diploid, triploid, and tetraploid cytotypes co-occurring throughout the Southern Appalachians. 
			Along with ongoing population genetic research, we investigated the cytogeography of <i>G. urceolata</i> in relation to stomatal cell size, soil chemistry, soil fungal 
			and bacterial community composition, and broad-scale climate. To hear more about this work, check out my <a href="https://lnkd.in/eyzvRWeY" target="_blank">talk at Evolution 2024.</a></h4>
		 <h4 style= "color: #000000; text-transform: none; font-weight: 350;"> <a style = "color: #fa4616;"><b>Methods:</b></a> Identifying ploidal diversity is a crucial first 
		 step to understanding the impact of WGD on patterns of biodiversity. To address shortcomings of available ploidal level inference methods, 
		 I developed the easy-to-use R package  <a href="https://mlgaynor.com/nQuack/" target="_blank">nQuack</a>. To hear more about this project, check out the second half of my <a href="https://lnkd.in/eGtJfBuQ" target="_blank">2024 Polyploidy Webinar.</a> </h4> 

	</div>

	<div class = "row">
	<div class="col-md-4">
		  		<div class="image-wrapper float-center pr-3" style="padding-top:30px;"><img  src="{{'/img/research/galax2021.jpeg' | relative_url}}" class="img-responsive" alt=""></div>
		  		<div class="caption"><i>Galax urceolata</i> (Diapensiaceae).</div>
	</div>

	</div>
	</div>
	
	<!-- Polyploid Paradigm -->
	<div class = "row">
		<div class="col-md-12">
		    	<h4 style= "color: #000000; text-transform: none; font-weight: 350;"><a style = "color: #fa4616;"><b>Phylogenetics and Beyond</b> </a> </h4>
		    		<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> To understand the long-term consequences of WGD on ecological and community dynamics, 
		    		phylogenetic-scale analyses are necessary.</h4>
					<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> At a community level, the effects of genome duplication remains unclear, therefore with 
					Drs. <a href="http://julienneng.weebly.com/" target="_blank">Julienne Ng</a> and <a href= "http://www.robertlaport.com/" target="_blank">Robert Laport</a> 
		    		we used a community phylogenetic approach to began to unravel the influence of genome duplication in communities of Rosaceae and Brassicaceae species. 
		    		We tested (1) whether polyploid species are more distantly related to diploids
		    	    within the same community than co-occurring diploids are to one another and (2) whether polyploid species tend to exhibit greater ecological success 
		    	    than diploid species. We did not find a consistent pattern, suggesting whole genome duplication impact on community structure may not be very black and white.
		  			</h4>
		</div>
		
	</div>

		
	
	</div>
	
		</div>
	</div>
</section>



<!-- Climatic Niche -->
<section id="climatic niche" style="padding-top: 30px; padding-bottom: 100px;">
	<div class = "container">
		<div class = "row">
   			<div class="caption" style="text-align:left; font-size: 22px; line-height: 1.5; color: #fa4616;"><b>Natural History Collections and Climatic Niche</b></div>
  		    <hr style = "margin-top: 10px; margin-bottom: 10px; border-top-width: 2px;">
   			<h4  style= "color: #000000; text-transform: none; font-weight: 350;"> Digitized biodiversity collections are an amazing resource for investigating climatic niche evolution and predicting how climate change will alter a species distribution. In conjunction with <a href="https://www.idigbio.org"/> iDigBio</a> and <a href="https://www.floridamuseum.ufl.edu/soltis-lab/">the Soltis lab</a>, I've contributed to and developed resources to aid in the use of digitized biodiversity data in research and teaching:   </h4>
			<br>

    <div class="col-md-4">
   		  		<div class="image-wrapper float-right pr-4" style="padding-right:30px"><img  src="{{'/img/research/HexTogether2024.png' | relative_url}}" class="img-responsive" alt=""></div>
    </div>

<div>	
			<ul  style="padding-top:30px; padding-left:20px">
			<li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/BiodiversityResources" target="_blank">Biodiversity Resource Round-up</a></li>
         <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://nataliepatten.github.io/gatoRs/index.html" target="_blank">gatoRs: Geographic and Taxonomic Occurrence R-Based Scrubbing R package</a>  (Also <a href="https://cloud.r-project.org/web/packages/gatoRs/index.html" target="_blank" > Available on CRAN<a>) </li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/soltislab/BotanyENMWorkshops" target="_blank">Using Digitized Collections-Based Data in Research, Botany 2020, 2021, 2022, and 2023</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/R4NaturalHistoryCollections-BCEENET2021" target="_blank">Using R for Digitized Natural History Collections (dNHC) in Research, BCEENET 2021 Virtual Meeting</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://doi.org/10.25334/84FC-TE88" target="_blank">Introduction to R with Biodiversity Data, QUBES Educational Resources</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://doi.org/10.25334/DRGD-F069" target="_blank">Cleaning Biodiversity Data: A Botanical Example Using Excel or RStudio, QUBES Educational Resources</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/CURE-FL-Plants" target="_blank">Course-based Undergraduate Research Experience: Florida Plants and Climate Change</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://biodiversity-specimen-data.github.io/specimen-data-use-case/" target="_blank"> API Office Hours with iDigBio API Working Group</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/ENMCrashCourse" target="_blank">Ecological Niche Modeling Crash Course</a></li>

        </ul>
		</div>
		</div>
	</div>
</section>
            
                	