---
layout: default_2
title: Research
permalink: research.html
background: '/img/bg-research.jpeg'
---

<!-- Introduction -->
<section id="research" style="padding-top: 30px; padding-bottom:25px;">
	<div class = "container">
		<div class="row">
			
		    <h4 style= "color: #000000; text-transform: none; font-weight: 350;"> I am a computational biologist with a strong background in data science and a growing understanding of bioinformatics. My research broadly encompasses evolution and ecology of flowering plants. Through my undergraduate and graduate research experience, I have focused on whole genome duplication, as well as population genetics and climatic niche evolution. I am also passionate about using natural history collections in research and education.</h4> 

		</div>
	</div>
</section>

<!-- Polyploidy Evolution -->
<section id="polyploidy evolution" style="padding-top: 20px; padding-bottom: 30px;">
	<div class = "container">
		<div class = "row">
   	    	<div class="caption" style="text-align:left; font-size: 22px; line-height: 1.5; color: #fa4616;"><b>Whole Genome Duplication</b></div>
    		<hr style = "margin-top: 10px; margin-bottom: 10px; border-top-width: 2px;">
    		<br>
		    <h4 style= "color: #000000; text-transform: none; font-weight: 350;"> My central research focus is currently on the cascading effects of whole genome duplication in flowering plants at multiple scales. Polyploidy, or whole genome duplication, is when additional copies of all chromosomes occur. Instead of having 2 copies of each chromosome (like humans), polyploids may have 3 to 96 copies of each chromosome (see 96 copies in <i>Ophioglossum reticulatum</i>). The extra set of chromosomes can be inherited from the same individual—this is known as autopolyploidy. </h4>		    		
					<br>
		
	<!-- Polyploid Paradigm -->
	<div class = "row">
		<div class="col-md-8">
		    	<h4 style= "color: #000000; text-transform: none; font-weight: 350;"><a style = "color: #fa4616;"><b>Polyploid Paradigm:</b> </a> Polyploidy, or whole genome duplication, is known to play an important role in the evolution and diversification of vascular plants. So far, most research has shown that whole genome duplication alone does not lead to a single set of expectations (other than maybe the nucleotypic effect), instead there are dynamic ecological or evolutionary consequences. The search for general patterns or trends associated with polyploidy events still remains – and this is a particularly interesting and fun topic to pursue. </h4>
		    		<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> At a community level, the effects of genome duplication remains unclear, therefore with Drs. <a href="http://julienneng.weebly.com/" target="_blank">Julienne Ng</a> and <a href= "http://www.robertlaport.com/" target="_blank">Robert Laport</a> 
		    		we used a community phylogenetic approach to began to unravel the influence of genome duplication in communities of Rosaceae and Brassicaceae species. We tested (1) whether polyploid species are more distantly related to diploids
		    	    within the same community than co-occurring diploids are to one another and (2) whether polyploid species tend to exhibit greater ecological success than diploid species. We did not find a consistent pattern, suggesting whole genome duplication impact on community structure may not be very black and white.
		  			</h4>
		  			<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> Zooming in, I am also interested in understanding the repercussions of whole genome duplication on cellular and gene level dynamics, since this would allow insight on the cascading ecological and evolutionary consequences associated with polyploidy. After a polyploidy event occurs, duplicated biosynthetic pathway genes may alter plant secondary metabolite production, thus influencing plant defense. With Simone Lim-Hing and Dr. Chase Mason, we investigated how polyploidy affects the composition and abundance of all classes of secondary metabolites in non-cultivated species. Again, we found that no consistent effect.
					</h4>
		</div>
		<div class="col-md-4">
		  		<div class="image-wrapper float-center pr-3" style="padding-top:30px;"><img  src="{{'/img/research/galax2021.jpeg' | relative_url}}" class="img-responsive" alt=""></div>
		  		<div class="caption"><i>Galax urceolata</i> (Diapensiaceae).</div>
		</div>
		
	</div>
					<br>
					<br>
					
	<!-- Mixed-ploidy species -->
	<div class = "row">


					<h4  style= "color: #000000; text-transform: none; font-weight: 350;"><a style = "color: #fa4616;"><b>Mixed-ploidy species:</b> </a> Historically, it was believed that a tetraploid had to outcompete its diploid progenitor to persist. The idea was that a tetraploid formed, spread, and caused the diploid to be geographically restricted and rare (Stebbins, 1947, 1971). We now know that closely related polyploids and their diploid progenitors can both remain extant.
		  			</h4>
		  			</div>
		  				<div class = "row">
		  				<div class="col-md-3">
		  	<div class="image-wrapper float-right pr-3"><img  src="{{'/img/research/Hex2024.png' | relative_url}}" class="img-responsive" alt=""></div>
		</div>
		  			<div class="col-md-9">
		  			<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> My dissertation centers on the persistence of mixed cytotype autopolyploid populations. My first two chapters focus on the theoretical ecological and genetic expectations during the formation, establishment, and persistence of mixed-cytotype populations. Check out my <a href="https://youtu.be/mEvSgXN6jbs?t=1990" target="_blank"> 2021 Polyploidy Seminar </a> on my second chapter. I currently have a <a href="https://doi.org/10.1101/2023.03.29.534764" target="_blank">preprint available for Chapter 1</a> and the R-based simulations can be found here: <a href="https://github.com/mgaynor1/AutoPop" target="_blank">mgaynor1/AutoPop</a>. </h4>
		  			<h4 style= "color: #000000; text-transform: none; font-weight: 350;">  I also developed an R package called nQuack which includes a modified statistical framework to predict ploidy level based on sequence data. </h4>
					<h4 style= "color: #000000; text-transform: none; font-weight: 350;"> My last two chapters focus on empirical case studies, focused on <i>Larrea tridentata</i> and <i>Galax urceolata</i></h4>
									
		</div>
			
	
	</div>
	
		</div>
	</div>
</section>

<!-- Population Genetics -->
<section id="other projects" style="padding-top: 20px; padding-bottom: 30px;">
	<div class = "container">
		<div class = "row">
   	    	<div class="caption" style="text-align:left; font-size: 22px; line-height: 1.5; color: #fa4616;"><b>Phylogenetic and Population Genetics</b></div>
			<hr style = "margin-top: 10px; margin-bottom: 10px; border-top-width: 2px;">
<div class = "row">
   <div class="col-md-8">
   
		 	<h4  style= "color: #000000; text-transform: none; font-weight: 350;"><a style = "color: #fa4616;"><b>Restoration Genetics:</b> </a> During my undergraduate, I worked with <a href="https://sciences.ucf.edu/biology/hoffman/">Drs. Eric Hoffman</a> and Linda Walters on a project assessing the ability of restoration efforts to maintain genetic diversity within restored populations of smooth cordgrass (Spartina alterniflora) within the Indian River Lagoon (IRL). We found that the current restoration efforts are able to maintain genetic diversity, therefore they ensure the evolutionary potential of these restoration populations.

            <br>
            <h4  style= "color: #000000; text-transform: none; font-weight: 350;"><a style = "color: #fa4616;"><b>Phylogenetics:</b></a> I am broadly interested in the maintenance of species diversity and I have utilized phylogenies to understand family or genus-level relationships and ecological dynamics including in the family Diapensiaceae. 

            <br>
            </h4>          
        </div>
            <div class="col-md-4">
		  		<div class="image-wrapper float-right"><img  src="{{'/img/research/pop.jpeg' | relative_url}}" class="img-responsive" alt=""></div>
		  		<div class="caption">A raccoon guarding a population of <i> Spartina alterniflora</i> </div>
		  	</div>       
		</div>
	</div>
</section>


<!-- Climatic Niche -->
<section id="climatic niche" style="padding-top: 30px; padding-bottom: 100px;">
	<div class = "container">
		<div class = "row">
   			<div class="caption" style="text-align:left; font-size: 22px; line-height: 1.5; color: #fa4616;"><b>Climatic Niche</b></div>
  		    <hr style = "margin-top: 10px; margin-bottom: 10px; border-top-width: 2px;">
   			<h4  style= "color: #000000; text-transform: none; font-weight: 350;"> Digitized biodiversity collections are an amazing resource for investigating climatic niche evolution and predicting how climate change will alter a species distribution. In conjunction with <a href="https://www.idigbio.org"/> iDigBio</a> and <a href="https://www.floridamuseum.ufl.edu/soltis-lab/">the Soltis lab</a>, I've contributed to and developed resources to aid in the use of digitized biodiversity data in research and teaching:   </h4>
			<br>

    <div class="col-md-4">
   		  		<div class="image-wrapper float-right pr-4" style="padding-right:30px"><img  src="{{'/img/research/HexTogether2024.png' | relative_url}}" class="img-responsive" alt=""></div>
    </div>

<div>	
			<ul  style="padding-top:30px; padding-left:20px">
			<li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/BiodiversityResources" target="_blank">Biodiversity Resource Round-up</a></li>
         <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://nataliepatten.github.io/gatoRs/index.html" target="_blank">gatoRs: Geographic and Taxonomic Occurrence R-Based Scrubbing R package</a>  (Also <a href="https://cloud.r-project.org/web/packages/gatoRs/index.html" target="_blank" > Available on CRAN<a>) </li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/soltislab/BotanyENMWorkshops" target="_blank">Using Digitized Collections-Based Data in Research, Botany 2020, 2021, 2022, and 2024</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/R4NaturalHistoryCollections-BCEENET2021" target="_blank">Using R for Digitized Natural History Collections (dNHC) in Research, BCEENET 2021 Virtual Meeting</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://doi.org/10.25334/84FC-TE88" target="_blank">Introduction to R with Biodiversity Data, QUBES Educational Resources</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://doi.org/10.25334/DRGD-F069" target="_blank">Cleaning Biodiversity Data: A Botanical Example Using Excel or RStudio, QUBES Educational Resources</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/CURE-FL-Plants" target="_blank">Course-based Undergraduate Research Experience: Florida Plants and Climate Change</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://biodiversity-specimen-data.github.io/specimen-data-use-case/" target="_blank"> API Office Hours with iDigBio API Working Group</a></li>
            <li style="font-size: 18px; text-transform: none; font-weight: 350;"><a href="https://github.com/mgaynor1/ENMCrashCourse" target="_blank">Ecological Niche Modeling Crash Course</a></li>

        </ul>
		</div>
		</div>
	</div>
</section>
            
                	